coding-the-legislation/README.md

```diff
@@ -7,6 +7,7 @@
    * [Vector calculus limitations](25_vectorial_computing.md)
    * [Case disjunction](30_case_disjunction.md)
    * [Periods](35_periods.md)
+   * [Inferences](37_inferences.md)
```

Comment:
If you don't also reference this new file in `SUMMARY.md`, the markdown won't be compiled.